test/pp_api_test/test_tokenizer_chat.cc

```diff
@@ -37,7 +37,7 @@ TEST(OrtxTokenizerTest, Phi4ChatTemplate) {
 
   auto err = OrtxApplyChatTemplate(
     tokenizer.get(), nullptr,
-    messages_json.c_str(), templated_text.ToBeAssigned(), true, false);
+    messages_json.c_str(), nullptr, templated_text.ToBeAssigned(), true, false);
```

Comment:
Can you add actual tests?

Reply:
oh yes, that was simply to ensure that previous C++ test does not break, will add testing in Python.